### ISSUE TITLE
immutable: resurrection of builder copy/from method

### ISSUE THREAD

**Issue by elucash** [immutables/immutables#87]
Resurrected "from" ( previously, it was named "copy") method on builder.
It was deleted last year in september-october.

Apparently, it's the most sound and powerful way to copy value into builder to produce modified value.
Usage of copyOf/withers with structural sharing copying may be either disabled (copy = false) or non desired.

The power comes from the way couple of objects may be merged, replacing regular values, accumulating collection values and replacing by non-empty optional values.

```java
/**
 * Adjust builder with values from provided {@link [type.typeAbstract.relative]} instance.
 * Regular attribute values will be overridden, i.e. replaced with ones of an instance.
 * Instance's absent optional values will not be copied (will not override current).
 * Collection elements and entries will be added, not replaced.
 * @param instance instance to copy values from
 * @return {@code this} builder for chained invocation
 */
```

73097c419eaacd164b5753ef30fe1359ca318ec6

**Comment by elucash** [comment]
```java
Value v1 = ..;

Value v2 = ImmutableValue.builder()
    .from(v1) // copy all attribute values from v1
    .attribute(a) // change some attribute value
    .build(); // build changed copies
```

**Comment by elucash** [comment]
Released in 2.0
